test/cbmc/proofs/MQTTAgent_Connect/MQTTAgent_Connect_harness.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * coreMQTT-Agent v1.0.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/* MQTT agent include. */
+#include "mqtt_agent.h"
+
+#include "mqtt_agent_cbmc_state.h"
+
+void harness()
+{
+    MQTTAgentContext_t * pMqttAgentContext;
+    MQTTAgentConnectArgs_t * pConnectArgs;
+    CommandInfo_t * pCommandInfo;
+
+    pMqttAgentContext = allocateMqttAgentContext( NULL );
+    __CPROVER_assume( isValidMqttAgentContext( pMqttAgentContext ) );
+
+    pConnectArgs = malloc( sizeof( MQTTAgentConnectArgs_t ) );
+    pCommandInfo = malloc( sizeof( CommandInfo_t ) );
```

Comment:
I think we will also need to assign a stub for `cmdCompleteCallback`

Reply:
Responded to #8